extensions/amp-social-share/1.0/social-share.js

```diff
@@ -54,31 +54,34 @@ export function SocialShare({
     checkedWidth,
     checkedHeight,
     checkedTarget,
+    errorState,
```

Comment:
Per discussion. I don't think we need to track `errorState`. We need to simplify it to a point of almost no error processing done in a component. The most important consequence is that we likely need to drop tests with `expect(() => mount(jsx)).to.throw`.

Reply:
Thanks @dvoytenko .  Updated revision is up.  I still included some minimal error checking which results in the component immediately returning `null` and does not render anything.

The corresponding tests check for an empty render instead of an error being thrown.

I have included a `console.warn` to notify the publisher and check for this in the tests.  These can be removed if you think it makes more sense.  Thanks!!